File: Örnek-6.3/Matlab/ucgenalan.m
a=input('Birinci kenar uzunluğunu giriniz (cm):');
b=input('İkinci kenar uzunluğunu giriniz (cm):');
aci=input('Arasındaki açıyı giriniz (derece):');

alan=a*b*sind(aci)/2;

fprintf('Üçgenin alanı (cm2): %0.3f',alan);

File: Örnek-6.3/C/ucgenalan.c
#include <stdio.h>
#include <stdlib.h>
#include <math.h>


float a,b,aci,alan;

int main() {
	
	printf("Bir kenarin uzunlugunu giriniz (cm): ");
	scanf("%f",&a);
	
	printf("Diger kenarin uzunlugunu giriniz (cm): ");
	scanf("%f",&b);
	
	printf("Arasindaki aciyi giriniz (derece): ");
	scanf("%f",&aci);
	
	alan=a*b*sin(M_PI*aci/180)/2;
	
	printf("Ucgenin alani (cm2): %0.2f",alan);
	

	return 0;
}
